lib/spack/spack/environment/__init__.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from .environment import (
+    Environment,
+    SpackEnvironmentError,
+    activate,
+    active,
+    active_environment,
+    all_environment_names,
+    all_environments,
+    config_dict,
+    create,
+    deactivate,
+    default_manifest_yaml,
+    default_view_name,
+    display_specs,
+    exists,
+    is_env_dir,
+    is_latest_format,
+    lockfile_name,
+    manifest_file,
+    manifest_name,
+    no_active_environment,
+    read,
+    root,
+    spack_env_var,
+    update_yaml,
+)
+
+__all__ = [
+    'activate',
+    'active_environment',
+    'active',
```

Comment:
Can we sort this one the same as isort did above?

Reply:
Oh, I thought isort handled this too with --fix, let me change it